### ISSUE TITLE
Allow configuration of caldav in GUI

### ISSUE THREAD

**Issue by Eeems** [tenJirka/reGenda#1]
It would be nice to be able to add the ability to set the URL, username, and password in the GUI instead of having to edit the configuration file on disk.

**Comment by tenJirka** [comment]
Agree, I'm considering if it won't be better wait until this [issue](https://github.com/rmkit-dev/rmkit/issues/202) will be solved, because I would like to add keyboard with support for Czech as well (that's problem when some letters will not be able to display) which will be needed for edit functions which I would like to add.
There are few symbols missing in Simple keyboard like `+`, `#`, `_` so I will probably design own keyboard anyway.

To sum it up,I plan to do implement it, but not sure which way chose.
